janusgraph-all/pom.xml

```diff
@@ -62,6 +67,16 @@
             <groupId>org.janusgraph</groupId>
             <artifactId>janusgraph-es</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.ow2.asm</groupId>
+                    <artifactId>asm</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.ow2.asm</groupId>
+                    <artifactId>asm-commons</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Thanks for the explanation. It looks like asm is no longer a dependency in ES so we'll be able to remove this when that's updated.